include/buildpack.bash

```diff
@@ -139,6 +139,12 @@ buildpack-list() {
 }
 
 buildpack-setup() {
+  # $import_path is defined in outer scope
```

Comment:
indentation looks off. perhaps tabs vs spaces here?

Reply:
Latest commit should address that